components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/internal/ServiceComponent.java

```diff
@@ -52,11 +55,34 @@
     protected void activate(ComponentContext ctxt) {
         try {
             setSecurityParams();
+            engagePoxSecurity();
         } catch (Throwable e) {
             log.error("Failed to activate Micro Integrator security bundle ", e);
         }
     }
 
+    private void engagePoxSecurity() {
+        try {
+            String enablePoxSecurity = CarbonServerConfigurationService.getInstance()
+                    .getFirstProperty("EnablePoxSecurity");
+            if (enablePoxSecurity == null || "true".equals(enablePoxSecurity)) {
+                AxisConfiguration mainAxisConfig = configCtx.getAxisConfiguration();
+                // Check for the module availability
+                if (mainAxisConfig.getModules().toString().contains(POX_SECURITY_MODULE)){
+                    mainAxisConfig.engageModule(POX_SECURITY_MODULE);
+                    log.info("UT Security is activated");
+                }
+                else {
+                    log.info("UT Security is not activated UTsecurity.mar is not available");
+                }
+            } else {
+                log.info("POX Security Disabled");
```

Comment:
Shall we make this as debug logs. If it is not enabled in carbon xml , we can silently ignore I think/

Reply:
fixed with ce0af4c